cdc/scheduler/internal/tp/replication_manager.go

```diff
@@ -376,25 +376,21 @@ func (r *replicationManager) AdvanceCheckpoint(
 	currentTables []model.TableID,
 ) (newCheckpointTs, newResolvedTs model.Ts) {
 	newCheckpointTs, newResolvedTs = math.MaxUint64, math.MaxUint64
```

Comment:
Can we return the current checkpoint if there is no table?

Reply:
I would prefer the current implementation.

If the current checkpoint is less than barrierTs, barrier won't be hit to take effect.